### PR TITLE
PLATO-496: Automated accessibility fixes for image group page

### PR DESCRIPTION
**Pull request by michael-iden**
Resolves [PLATO-496](https://jira.jstor.org/browse/PLATO-496)

Fixes automated accessibility errors generated from the QA gem.

### CODE REVIEW

File: src/app/asset-grid/asset-grid.component.pug [hunk 1]
```diff
@@ -123,9 +123,9 @@
                 [largeThmbView]="largeThmbView"
               )
             .mobile--hide
-              input.check(tabIndex="3", type="checkbox", [checked]="isSelectedAsset(asset) > -1", [ngStyle]="{ 'opacity': isSelectedAsset(asset) > -1 ? '1' : '0' }", (change)="editMode = true; selectAsset(asset);")
+              input.check([attr.aria-label]="'Select asset \"' + asset.name + '\"'", tabIndex="3", type="checkbox", [checked]="isSelectedAsset(asset) > -1", [ngStyle]="{ 'opacity': isSelectedAsset(asset) > -1 ? '1' : '0' }", (change)="editMode = true; selectAsset(asset);")
             .mobile--show
-              input.check-mobile(type="checkbox", [checked]="isSelectedAsset(asset) > -1", (change)="editMode = true; selectAsset(asset);")
```
Comment (michael-iden) [inline src/app/asset-grid/asset-grid.component.pug]: This is real ugly. Please lmk if there is a better way to do the label construction.

Comment (eslawski) [reply src/app/asset-grid/asset-grid.component.pug]: Would it be overkill to extract the string building into a function/variable on `asset-grid.component`? Might be able to use ES6 string templates there too?

Comment (gbassham) [reply src/app/asset-grid/asset-grid.component.pug]: I think Evan is right, I would put this into a function. Easier to maintain if it has to be changed in the future.

Comment (michael-iden) [reply src/app/asset-grid/asset-grid.component.pug]: Done!

**Pull request merged**